kubernetes/helm/values.override_production.auction-ui.k8s.sidestream.tech.yaml

```diff
@@ -0,0 +1,22 @@
+# Value overrides for the deployment to the following host:
+hosts:
+  - "production.auction-ui.k8s.sidestream.tech"
+  - "auctions.makerdao.network"
+
+ingress:
+  isCertManagerActivated: true
+  # We can only generate a TLS cert for our own domain,
+  # maker needs to take over their domains TLS termination
+  tls:
+    - hosts:
+        - "production.auction-ui.k8s.sidestream.tech"
+      secretName: "auction-ui-tls-certificate"
```

Comment:
So we would still need to redeploy it, since tls manager will not run itself when dns entry has changed (and therefore requiring manual edit of this file)?

Reply:
Same with the `unified-auctions.makerdao.com` domain, I guess

Reply:
> So we would still need to redeploy it, since tls manager will not run itself when dns entry has changed (and therefore requiring manual edit of this file)?

No, unless I misunderstood, but: What we're doing now is listening to two hosts:
- production.auction-ui.k8s.sidestream.tech, with the TLS cert stored in `auction-ui-tls-certificate`
- auctions.makerdao.network, without any TLS cert

They have to take care of TLS termination (this is also what's going on right now with the current deployment).

Reply:
> Same with the unified-auctions.makerdao.com domain, I guess

This is the first time I head about this domain, but yeah, this one is not currently listened for.

Reply:
> They have to take care of TLS termination

I was not aware of that!

> This is the first time I head about this domain

I've mentioned it here: https://github.com/sidestream-tech/unified-auctions-ui/pull/57#discussion_r793443121
Do you want to add it in the same PR before merging it?

Reply:
So, I've added the second domain - when we change the main environment to `unified-auctions.makerdao.com` we will have to update the deployment anyways though, to update the plausible link. But I guess that this is not super relevant, as we can just count both `unified-auctions.makerdao.com` and `auctions.makerdao.network` as production for analytics.

Reply:
> plausible link

It's not a link, it's something like a project id! Maybe we should change the variable name to make it less confusing 😕